install-raspbian.sh

```diff
@@ -67,7 +67,8 @@ apt update
 apt dist-upgrade -y
 
 info "### Photobooth needs some software to run."
-apt install -y libapache2-mod-php php-gd gphoto2 unclutter
+apt install -y nginx php-fpm php-gd gphoto2 unclutter
+/etc/init.d/nginx start
```

Comment:
`systemctl start nginx` should be used but it's not necessary here as the service will be automatically started.